client/src/components/App.jsx

```diff
@@ -0,0 +1,63 @@
+import React, {useState, useEffect} from 'react';
+import styled from 'styled-components';
+import MainPage from './MainPage';
+import Login from './Login'
+import axios from 'axios';
+
+const App = () => {
+
+  const [isLoggedIn, setIsLoggedIn] = useState(false);
+  const [loginUserName, setLoginUserName] = useState('');
+
+  useEffect(() => {
+    axios.get('/api/login').then(res => setIsLoggedIn(res.data))
+  }, [])
+
+  const logout = () => {
+    axios.delete('/api/logout').then(res => setIsLoggedIn(res.data)).then(console.log('isLoggedIn:', isLoggedIn))
```

Comment:
I'm not sure it makes sense to use a DELETE request here. I'd probably just use a POST. DELETE implies you're removing the resource referenced by the path, i.e. you're deleting `/api/logout` which doesn't quite make sense.